tests/encodeinput.cpp

```diff
@@ -31,6 +31,52 @@
 
 using namespace YamiMediaCodec;
 
+VideoRateControl string_to_rc_mode(char* str)
+{
+    VideoRateControl rcMode;
+
+    if (!strcasecmp(str, "CBR"))
+        rcMode = RATE_CONTROL_CBR;
+    else if (!strcasecmp(str, "VBR"))
+        rcMode = RATE_CONTROL_VBR;
+    else if (!strcasecmp(str, "CQP"))
+        rcMode = RATE_CONTROL_CQP;
+    else {
+        printf("Unsupport  RC mode\n");
+        rcMode = RATE_CONTROL_NONE;
+    }
+    return rcMode;
+}
+
+VAProfile string_to_profile(char* str)
+{
+    VAProfile vaProfile;
+
+    if (!strcasecmp(str, "Baseline"))
```

Comment:
since you add this profile, did you check it all? to make sure it all decodable by intel video tool and no errors?